foundations/javascript_basics/variables_and_operators.md

```diff
@@ -1,86 +1,128 @@
 
 ```html
-  <script src="javascript.js"></script>
+<script src="javascript.js"></script>
 ```
 
 JavaScript files have the extension `.js` similar to `.css` for stylesheets. External JavaScript files are used for more complex scripts.
 
+We named our file `javascript.js` but we could have chosen any name like `my-script.js` or even no name `.js`. What is really important is the `.js` extension.
+
 ### Variables
 
+These are the building blocks of any program. As an abstract example, let's consider a program that takes in two numbers and returns the sum of those numbers.
+
+It could store the first number as a variable, `a` and the second number as another variable `b`. Then it would output their sum - `a + b`.
+
 You can think of variables as "storage containers" for data in your code.
 
 ![Variable Box Illustration](https://cdn.statically.io/gh/TheOdinProject/curriculum/d39eaf2ca95e80705f703bb218216c10508f5047/foundations/javascript_basics/fundamentals-1/imgs/00.png)
 
-<span id="variable-declaration">Until recently there was only one way to create a variable in JavaScript &mdash; the `var` statement. But in the newest JavaScript versions we have two more ways &mdash; `let` and `const`.</span>
+You can declare variables using the `let` keyword. Lets try it! (No pun intended).
+
+```javascript
+let name = "John";
+let surname = "Doe";
+
+console.log(name);
+console.log(surname);
+```
+
+What will the `console.log` output? Try it out!
+
+You can also re-assign variables:
+
+```javascript
+let age = 11;
+console.log(age); // outputs 11 to the console
+
+age = 54;
+ 
+console.log(age); // what will be output now?
+```
+
+Notice the lack of `let` on line 4 - we don't need it since the variable has already been *declared* earlier and we are just re-assigning it here!
+
+Re-assigning is cool and all, but what if we *don't* want it to happen? For example we might have a *constant* `pi` which will never change. We can accomplish this using the `const` keyword.
+
+```javascript
+const pi = 3.14;
+pi = 10;
+
+console.log(pi); // What will be output?
+```
+
+Your intuition may tell you that `3.14` will be output. Try it!
+
+An error is thrown. It doesn't even reach the `console.log`! You may wonder why we would *want* an error in our code. Truth be told, errors are incredibly helpful at telling us what is wrong with our code and exactly where the issue is. Without them, our code would still not do what we may want it to, but it would be a major pain to try and find what's wrong!
+
+So in summary, there are two ways to declare a variable:
 
-1. This tutorial on [JavaScript variables](http://javascript.info/variables) will explain everything you need to know! Be sure to do the **Tasks** at the end. Information won't stick without practice!
+- With `let`, which we can re-assign.
+- Using `const` which we **can't** re-assign and will throw an error if we try.
```

Comment:
Pick one word, "with" or "using", or just omit both.

Reply:
I agree, it looks cleaner without those filler words